src/utils/languages.ts

```diff
@@ -17,6 +17,27 @@ export const availableLanguages: Language[] = [
   { code: 'ja', name: '日本語', flag: '🇯🇵' },
   { code: 'ru', name: 'Русский', flag: '🇷🇺' },
   { code: 'th', name: 'ไทย', flag: '🇹🇭' },
+  { code: 'en', name: 'English', flag: '🇺🇸' },
```

Comment:
In this project, availableLanguages is used directly for the selection items.

https://github.com/miurla/babyagi-ui/blob/main/src/components/Sidebar/SidebarSettings.tsx#L175

So I’d like to sort this list in ascending order by language code.
Alternatively, the sorting logic can be added on the component side.

Reply:
Instead of importing the availableLanguages you can import the languages which is located in this file. That is an alphabetically ASC sorted array of the availableLanguages. I'm going to change it in this PR if you don't mind.

Reply:
I missed the `languages` 🙏. 
It would be great if you could change it here.